hi-ml-cpath/src/health_cpath/configs/classification/DeepSMILEPanda.py

```diff
@@ -87,20 +88,21 @@ def get_data_module(self) -> PandaTilesDataModule:
             crossval_index=self.crossval_index,
             dataloader_kwargs=self.get_dataloader_kwargs(),
             seed=self.get_effective_random_seed(),
+            pl_replace_sampler_ddp=self.pl_replace_sampler_ddp,
         )
 
     def get_slides_dataset(self) -> Optional[PandaDataset]:
         return PandaDataset(root=self.local_datasets[1])                             # type: ignore
 
     def get_test_plot_options(self) -> Set[PlotOption]:
         plot_options = super().get_test_plot_options()
-        plot_options.add(PlotOption.SLIDE_THUMBNAIL_HEATMAP)
+        plot_options.update([PlotOption.SLIDE_THUMBNAIL, PlotOption.ATTENTION_HEATMAP])
         return plot_options
 
 
 class TilesPandaImageNetMIL(DeepSMILETilesPanda):
     def __init__(self, **kwargs: Any) -> None:
-        super().__init__(encoder_type=ImageNetEncoder.__name__, **kwargs)
+        super().__init__(encoder_type=Resnet18.__name__, **kwargs)
```

Comment:
Why is a different encoder type used here?

Reply:
You can find more details here https://github.com/microsoft/hi-ml/pull/574/files#diff-cbdcc375c22df662ad8be1144d92982cfa401630cfa7bc81f39206b3574aa857 
We renamed the encoders to specify the Architecture